vault/identity_store_oidc.go

```diff
@@ -1032,6 +1042,9 @@ func (k *namedKey) generateAndSetNextKey(ctx context.Context, logger hclog.Logge
 }
 
 func (k *namedKey) signPayload(payload []byte) (string, error) {
+	if k.SigningKey == nil {
+		return "", fmt.Errorf("signing key is nil")
```

Comment:
nit: I think a call to the rotate API would get users out of this situation? I wonder if we can make this error message more actionable. For example, adding something like "rotate key <name> to generate a signing key" to the end of the message above.